### PR TITLE
restructure front page

### PR DESCRIPTION
**Pull request by critzo**
@georgiamoon this PR is now ready for your review. Changes are all previewable: https://critzo.github.io/m-lab.github.io/

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/389)

### CODE REVIEW

File: _sass/layouts/_header.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 header.header-section {
   zoom: 1;
-  margin: 30px 0px;
+  margin: 14px 0px;
```
Comment (georgiamoon) [inline _sass/layouts/_header.scss]: Let's remove this and put back the older header spacing. (I think this is the header area spacing, if that's wrong, then disregard, and let's edit the right margin/padding)

File: _sass/layouts/_homepage.scss [hunk 2]
```diff
@@ -132,11 +132,56 @@
   }
 }
 
+// Mission & Tag Line
+.tagline-section {
+  width: 100%;
+  background: #81addd;
+  padding-bottom: .5em;
+  padding-top: .5em;
```
Comment (georgiamoon) [inline _sass/layouts/_homepage.scss]: Can we up these to give the tagline section a bit more space?

**Comment by critzo** [comment]
Thanks for these comments @georgiamoon . I'll make another update a little later and ping you back.

**Comment by critzo** [comment]
@georgiamoon I've made some more adjustments based on your feedback. Please take a look when you have a sec? https://critzo.github.io/m-lab.github.io/

**Comment by georgiamoon** [comment]
I think this looks great! 

My two tiny tweaks:
- Let's remove the "Mission Statement" heading
- In the header with the logo and the "MeasurementLab is a partnership..." let's leave the old spacing, which had the text and logo further apart.

Thanks for working on this!

-Georgia

**Comment by critzo** [comment]
Sounds great! Thanks for the review! Good to get that done!

**Pull request merged**